app/presenters/rejection_reasons/rejection_reasons_presenter.rb

```diff
@@ -31,10 +32,10 @@ def reasons
     def nested_reasons(reason)
       reason.selected_reasons.each_with_object([]) do |nested_reason, ary|
         if nested_reason.details
-          ary << "#{nested_reason.label}:" if render_label?(nested_reason.label, reason.selected_reasons)
+          ary << "#{nested_reason.label_text}:" if render_label?(nested_reason.label, reason.selected_reasons)
```

Comment:
```suggestion
          ary << "#{nested_reason.label_text}:" if render_label?(nested_reason.label_text, reason.selected_reasons)
```
 are we changing the method name from label to label_text? I'm not sure what this change is for.

Reply:
Yes, because on different pages the rejection reason will read differently and today there is no way to separate the config, from the database, from the different pages and different interfaces. so I implemented the `label_text` that will look replace the reason label or returns the default from config.

Reply:
And should it be the label or the label_text that gets passed to the `render_label?` method? This feels very fragile

Reply:
As this is a new method on the reason model, it might be useful to see a spec describing it `reason_spec` so it is clear how it should be used and in what circumstances.

Reply:
I totally understand the concerns about fragility—it’s definitely a tricky area! 😞😞😞😞😞

The current structure for rejection reasons does make it challenging to extend things smoothly. That’s why I opted for this approach; it seemed like the best way to differentiate between config, database, and presentation without having to rewrite a ton of code.

Although I totally recommend reviewing the whole rejection reasons approach going forward to make everything more simpler and more extendable. If in the future we would need to show the same reason in different way, it shouldn't be that difficult.

I've pushed the spec on 8e827b1813ad7ca7bbff7be28a07dacaa82bb195